Dockerfile.canary

```diff
@@ -1,15 +1,19 @@
 FROM node:20-bookworm
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+
 RUN apt update && apt install -y awscli
 
 WORKDIR /src
 
 COPY ../ ./
-RUN npm ci
-RUN npm run build
+RUN pnpm install
```

Comment:
should this be a clean install too?

Reply:
Wdym clean install? 🤔

Reply:
`npm ci` does a clean install in CI envs. Wondering if we need to do the same (-i think it's `pnpm --fronzen-lockfile`)